README.md

```diff
@@ -1,3 +1,77 @@
-# tensorbay-python-sdk
+# TensorBay Python SDK
 
-Graviti TensorBay Python SDK
+TensorBay Python SDK is a python library to access TensorBay and manage your datasets.  
+It provides:
+
+-   A pythonic way to access your TensorBay resources by TensorBay OpenAPI.
+-   An easy-to-use CLI tool `gas` (Graviti AI service) to communicate with TensorBay.
```

Comment:
How about adding a hyperlink to GAS?

Reply:
The link can be found in the right side of the Github page